<launch>

    <node name="tracking_control_antsmc" pkg="sensors" type="tracking_control_antsmc">
    <param name = "k_x" value = "0.05" />
	<param name = "k_y" value = "0.05" />
	<param name = "kmin_x" value = "0.01" />
	<param name = "kmin_y" value = "0.01" />
	<param name = "k2_x" value = "0.01" />
    <param name = "k2_y" value = "0.01" />
	<param name = "mu_x" value = "0.1" />
	<param name = "mu_y" value = "0.1" />
    <param name = "beta_x" value = "1.0" />
	<param name = "beta_y" value = "1.0" />
    <param name = "gamma_x" value = "1.6" />
	<param name = "gamma_y" value = "1.6" />
    </node>

</launch>
